content/blog/radicle-architecture.md

```diff
@@ -1,15 +1,14 @@
 ---
 
-title: "Radicle"
+title: "Radicle Architecture"
 date: 2019-03-28
 markup: mmark
 author: "Julian Arni and James Haydon"
 draft: false
+summary: "Radicle is a system for code collaboration that has several advantages over
```

Comment:
How about: "In this post we explain the P2P architecture underlying Radicle: decentralised state machines stored on IPFS."

@jkarni

Reply:
@juliendonck I also wouldn't mind just having the first part repeat itself and not call it summary but sneap peak.A distinct summary could even be better I agree.

Reply:
I agree with @jameshaydon's summary.